app/src/test/java/com/example/kotlincalculator/model/CalculatorModelTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.kotlincalculator.model
+
+import com.example.kotlincalculator.mvp.model.CalculatorModel
+import com.example.kotlincalculator.utils.EMPTY_STRING
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+
+class CalculatorModelTest {
+
+    companion object {
+        private const val ZERO: String = "0"
+        private const val ONE: String = "1"
+        private const val PLUS: String = "+"
+    }
+
+    private var modelTest = CalculatorModel()
+
+    @Test
+    fun clearVisorTest() {
+        modelTest.cleanVisor()
+        assertEquals(EMPTY_STRING, modelTest.firstOperand)
+        assertEquals(EMPTY_STRING, modelTest.secondOperand)
+        assertEquals(EMPTY_STRING, modelTest.operator)
+    }
+
+    @Test
```

Comment:
no need to test the Kotlin API, please test only the public methods that contains logic

Reply:
done